tests/unittests/test_ds_identify.py

```diff
@@ -363,8 +366,12 @@ def write_mock(data):
             out = e.stdout
             err = e.stderr
 
+        runpath = "run"
```

Comment:
nit: Could we set this variable near the other changes and include a comment explaining what it is for?

```diff
diff --git a/tests/unittests/test_ds_identify.py b/tests/unittests/test_ds_identify.py
index 4e8126ba6..1bc9dc42a 100644
--- a/tests/unittests/test_ds_identify.py
+++ b/tests/unittests/test_ds_identify.py
@@ -340,11 +340,16 @@ class DsIdentifyBase(CiTestCase):
         ]
 
         uname = "Linux"
+        runpath = "run"
         written = []
         for d in mocks:
             written.append(d["name"])
             if d["name"] == "uname":
                 uname = d["out"].split(" ")[0]
+        # set runpath so that BSDs use /var/run rather than /run
+        if uname != "Linux":
+            runpath = "var/run"
+
         for data in mocks:
             mocklines.append(write_mock(data))
         for d in default_mocks:
@@ -366,10 +371,6 @@ class DsIdentifyBase(CiTestCase):
             out = e.stdout
             err = e.stderr
 
-        runpath = "run"
-        if uname != "Linux":
-            runpath = "var/run"
-
         cfg = None
         cfg_out = os.path.join(rootd, runpath, "cloud-init/cloud.cfg")
         if os.path.exists(cfg_out):
```

Reply:
yeah that reads more sensible